plugin/src/main/kotlin/com/facebook/testing/screenshot/build/ScreenshotsPlugin.kt

```diff
@@ -71,11 +71,19 @@ class ScreenshotsPlugin : Plugin<Project> {
   private fun generateTasksFor(project: Project, variant: TestVariant) {
     variant.outputs.all {
       if (it is ApkVariantOutput) {
+        val cleanScreenshots =createTask(
+            project,
+            CleanScreenshotsTask.taskName(variant),
```

Comment:
`CleanScreenshotsTask` is missing from the PR

Reply:
woops

Reply:
fixed